components/StoryblokProvider.tsx

```diff
@@ -58,12 +58,16 @@ const components = {
 };
 
 storyblokInit({
-  accessToken: process.env.STORYBLOK_ACCESS_TOKEN,
+  accessToken: process.env.NEXT_PUBLIC_STORYBLOK_ACCESS_TOKEN, // Public token because this is in client side code.
```

Comment:
Layout is always a client component. I don't feel good about having a token in the rendered codebase  but at least it is just public data. Storyblok's packages all revolve around the storyblokInit function so I don't see an easy way out of this right now. 

Thought: Maybe I can pull it from `window.location`?